lib/cc/engine/analyzers/parser_base.rb

```diff
@@ -0,0 +1,13 @@
+module CC
+  module Engine
+    module Analyzers
+      class ParserBase
+        protected
```

Comment:
I think I do this habitually more for how it reads than how it actually works. But good point, changing.